README.md

```diff
@@ -358,63 +356,64 @@ but use our tarball in place of theirs.
 1. Zero the beginning of the SD card
 
 ```
-# dd if=/dev/zero of=/dev/sdX bs=1M count=32
```

Comment:
this one. it depends on how the uSD is connected. I was doing it with a card reader so it showed up as /dev/sdb

Reply:
I thought mmcblk would hint in the right direction and prevent wrong targets. But both are okay.